compose-development.yml

```diff
@@ -33,19 +35,23 @@ webappdev:
   links:
     - rabbitmqdev:rabbit
     - databasedev
-    - emaildev
     - celerydev
   environment:
     - DJANGO_CELERY_BROKER_URL=amqp://guest@rabbit
     - DJANGO_DATABASE_URL=postgis://postgres@databasedev/postgres
     - DJANGO_DEBUG=true
-    - DJANGO_EMAIL_HOST=emaildev
-
-
-rabbitmqdev:
-  extends:
-    file: compose-common.yml
-    service: rabbitmqbase
+    - DJANGO_EMAIL_FILE_PATH=/dev-emails
+    - DJANGO_EMAIL_BACKEND=django.core.mail.backends.filebased.EmailBackend
+    - DJANGO_EMAIL_HOST=localhost
+    - DJANGO_EMAIL_HOST_PASSWORD=''
+    - DJANGO_EMAIL_HOST_USER=''
+    - DJANGO_EMAIL_PORT=25
```

Comment:
Do we need all of `DJANGO_EMAIL_HOST`, `DJANGO_EMAIL_HOST_PASSWORD`, `DJANGO_EMAIL_HOST_USER`, `DJANGO_EMAIL_PORT` when using `DJANGO_EMAIL_BACKEND=django.core.mail.backends.filebased.EmailBackend`?